XamlControlsGallery/ControlPages/ListViewPage.xaml.cs

```diff
@@ -242,6 +251,53 @@ private void StackPanel_loaded(object sender, RoutedEventArgs e)
             stackPanelObj = sender as ItemsStackPanel;
         }
 
+        //===================================================================================================================
+        // Filtered List Example
+        //===================================================================================================================
```

Comment:
I really love this comments/blocks, they make it definitely easier to read the code!

Reply:
:)